2025-06-02 20:07:12.265957 | failing_func error: ValueError | Execution time: 0.0000s | Inputs: (1, 2), {} | Error: Test error
